ops/opset13/cos_test.go

```diff
@@ -0,0 +1,99 @@
+package opset13
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/advancedclimatesystems/gonnx/ops"
+	"github.com/stretchr/testify/assert"
+	"gorgonia.org/tensor"
+)
+
+func TestCosInit(t *testing.T) {
+	c := &Cos{}
+
+	// since 'cos' does not have any attributes we pass in nil. This should not
+	// fail initializing the cos.
+	err := c.Init(nil)
+	assert.Nil(t, err)
+}
+
+func TestCos(t *testing.T) {
+	tests := []struct {
+		cos      *Cos
+		backing  []float32
+		shape    []int
+		expected []float32
+	}{
+		{
+			&Cos{},
```

Comment:
use the `newCos` method? Or is this the way we do it for other operators as wel?

Reply:
This is how we do it for all operators now.